package.json

```diff
@@ -83,11 +83,11 @@
     "react2angular": "4.0.2",
     "readable-stream": "2.3.3",
     "redux": "3.5.2",
-    "rendition": "4.27.3",
+    "rendition": "4.41.1",
```

Comment:
Is this update intended?

Reply:
Yes, it fixes an issue with an internal rendition package that bumped version and changed the folder structure which broke the builds